release/packbeam.in

```diff
@@ -19,4 +19,10 @@
 # SPDX-License-Identifier: Apache-2.0 OR LGPL-2.1-or-later
 #
 
-exec PACKBEAM_INSTALL_LOCATION/bin/packbeam "$@"
+root_dir="$(cd $(dirname $0)/.. && pwd)"
+pkg_root="${root_dir}/atomvm_packbeam"
+
+PATH="${pkg_root}/erts-{{ release_erts_version }}/bin:${PATH}"
+export PATH
```

Comment:
What ends up in the `${pkg_root}/errs-{{ release_erts_version }}/bin` directory? And why does it need to be exported?

Reply:
Escript begin with `/usr/bin/env escript`, so escript needs to be in the PATH.  Since this is a release with OTP embedded, it uses relx overlays to get the correct path to the current ERTS version that is shipped in the release.

Reply:
Thanks for the clarification, I was a little confused by the next line `exec ${pkg-root}/bin/packbeam`. I thought maybe there was old code left.